server/server.ts

```diff
@@ -301,8 +331,22 @@ function tick(game: InternalState, deltaMs: number) {
     if (player.dashCooldown && player.dashCooldown < Date.now()) {
       player.dashCooldown = undefined;
     }
+
+    if (player.score > highScore) {
+      highScore = player.score;
+    }
   });
 
+  if (highScore >= lobbyInfo.state.winningScore) {}
+  const newState: LobbyState =
+    lobbyInfo.state != null
+      ? lobbyInfo.state
+      : {
+        playerCount: game.players.length + 1,
+      };
+  newState.playerCount = game.players.length + 1;
```

Comment:
why are we incrementing the player count here? seems like running this function on tick would cause the player count to explode

Reply:
This block was added by accident, I have removed!